examples/hosts/app-integration/external-data/tests/taskList.test.ts

```diff
@@ -20,6 +20,6 @@ describe("taskList", () => {
 
     it("loads and there's an input", async () => {
         // Validate the input shows up
-        await page.waitForSelector("input");
+        await page.waitForSelector("input", {timeout: 12_000});
```

Comment:
Is this still needed?

Reply:
Note that the default test timeout is 5000 ms, so the test itself will timeout before this is allowed to finish the full 12000. Might be worth adding this to the test too while you're working through issues with it.